packages/@guardian/source-react-components/src/brand/SvgGuardianLogo.tsx

```diff
@@ -34,14 +34,24 @@ export const SvgGuardianLogo = ({
 		<>
 			<svg
 				xmlns="http://www.w3.org/2000/svg"
-				viewBox="0 0 300 100"
+				viewBox="0 -3 300 100"
```

Comment:
Any context on the -3?

Reply:
I used [this Figma source](https://www.figma.com/file/HH2LEmJBGHdjCZdrwcm4Io/%E2%97%90-Brand?node-id=1%3A86) to get the original paths:

<img width="493" alt="image" src="https://user-images.githubusercontent.com/76776/196377941-3a51ed93-1810-4060-93c3-3f7ad908d13e.png">

As you can see, the logo’s bounding box is not 300×100, but a shy 97. For resizing purposes, I think it is much preferable to have a consistent ratio between the width and height of your assets, so that you get entire pixels on both axis–half would be 150×50, fourth would be 75×25, etc. When the previous asset was prepared, it was aligned to the bottom of this bounding box, so I had to shift the artboard 3 pixels up. That’s what the first two values of `viewBox` are for.